qa/L0_infer/infer_test.py

```diff
@@ -47,7 +47,7 @@
 np_dtype_string = np.dtype(object)
 
 
-class InferTest(unittest.TestCase):
+class InferTest(tu.TestResCollector):
```

Comment:
Shouldn't the unittest import no longer be needed after this change?

Reply:
Sorry I missed this comment. You're right, I will remove those imports.

Reply:
It is required in the end of files for running the unittest.